testsuite/dockerfiles/server-all-in-one-dev/Dockerfile

```diff
@@ -2,14 +2,14 @@ ARG BASE
 ARG VERSION
 FROM ${BASE}:${VERSION}
 
-RUN zypper ref && \
+RUN zypper -n --gpg-auto-import-keys ref && \
```

Comment:
Nice. I forgot to tell you about that, too. Sorry. But I also had it in my previous runs. See https://github.com/nodeg/uyuni/actions/runs/10472171110/job/29001012967